ztp/source-crs/validatorCRs/informDuValidator.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfigPool
 metadata:
   name: $mcp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "1000000000000"
```

Comment:
Maybe something not quite so huge? 10000?

Reply:
Sure, I know it's a little bit crazy :)